package.json

```diff
@@ -37,8 +37,8 @@
 		"url": "https://github.com/pact-foundation/pact-node/issues"
 	},
 	"dependencies": {
-		"@pact-foundation/pact-mock-service": "~2.1.0",
-		"@pact-foundation/pact-provider-verifier": "~1.1.1",
+		"@pact-foundation/pact-mock-service": "^2.1.0",
```

Comment:
Up to you - it does cause a bit of a maintenance nightmare but the upside is control. Sure - want me to update?